gpt-survey-summarizer/values.yaml

```diff
@@ -60,7 +60,7 @@ image:
   # -- The pullPolicy used when pulling the image
   pullPolicy: IfNotPresent
   # -- The tag of the iamge. Overrides the image tag whose default is the chart appVersion.
-  tag: "0.2.0-4a6bee4"
+  tag: "0.2.0"
```

Comment:
**suggestion:** I think we should change this to `""` (aka `latest`), because:
- we don't have any genuine value to set as default,
- in case of ArgoCD managed, the image tag will be specified in gitops repo and override the default tag,
- in case of Keel managed, the deployment would be patched by Keel properly.

Also, if we manually deploy with `helmfile template | kubectl apply -f -`, or if ArgoCD sync does not respect the ignored diff and start a synchronisation (which is a bug we have at the moment), that would deploy a non-existing docker image, and therefore K8s deployment would not stop the running production.

```suggestion
  tag: ""
```